app/components/environment/environment-service.es6

```diff
@@ -179,22 +179,33 @@ class Environment {
         let protocol = 'https://';
 
         let prefix;
-        if ( /QA/.test(env) ) {
+        if ( /SA QA/.test(env) ) {
```

Comment:
I didn't mention this in person but in the honeydew project, `dashboard-search.js` incorrectly assumed that the env (`QA` or `Stage`) would be the first part of this string. It's cool if that's no longer the case, but just noting that it'll require matching changes in `dashboard-search.js`.

https://git.alias.sharecare.com/QA/honeydew/blob/Get-PS-Build-Numers/honeydew-ui/htdocs/js/dashboard-search.js#L64-70